src/seedu/addressbook/data/person/Name.java

```diff
@@ -42,7 +42,12 @@ public static boolean isValidName(String test) {
     public List<String> getWordsInName() {
         return Arrays.asList(fullName.split("\\s+"));
     }
-
+	
```

Comment:
Nice simple solution to enhance the functionality of the find function. Great job!